.github/workflows/docs.yml

```diff
@@ -0,0 +1,40 @@
+name: Documentation
+
+on: [push]
+
+jobs:
+  build-linux:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 5
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    - name: Add conda to system path
+      run: |
+        # $CONDA is an environment variable pointing to the root of the miniconda directory
+        echo $CONDA/bin >> $GITHUB_PATH
+    - name: Install dependencies
+      env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think this may need to be
```
          GH_TOKEN: ${{ secrets.RMG_DEV_TOKEN }}
```
based on https://github.com/ReactionMechanismGenerator/RMG-Py/blob/master/.github/workflows/CI.yml

Reply:
replaced it!